btcec/signature.go

```diff
@@ -206,13 +109,6 @@ func ParseSignature(sigStr []byte, curve elliptic.Curve) (*Signature, error) {
 	return parseSig(sigStr, curve, false)
 }
 
-// ParseDERSignature parses a signature in DER format for the curve type
-// `curve` into a Signature type.  If parsing according to the less strict
-// BER format is needed, use ParseSignature.
-func ParseDERSignature(sigStr []byte, curve elliptic.Curve) (*Signature, error) {
```

Comment:
It makes sense to delete this but can you update all comments of exported methods that call `parseSig`? E.g., `ParseSignature` still speaks of the `BER` format which isn't used anymore.